themes/src/main/resources/theme/keycloak.v2/account/src/app/content/my-resources-page/PermissionRequest.tsx

```diff
@@ -48,7 +49,7 @@ interface PermissionRequestState {
 }
 
 export class PermissionRequest extends React.Component<PermissionRequestProps, PermissionRequestState> {
-    protected static defaultProps = { permissions: [], row: 0 };
+    protected static defaultProps:any = { permissions: [], row: 0 };
```

Comment:
Can you add a correct type her instead of `any`?

Reply:
New type created for Permissions, used here. and in one additional instance.